### ISSUE TITLE
Question about out of scope parameter

### ISSUE THREAD

**Issue by DorianDe** [wit-ai/wit#2007]
Hi, I am running into some issues with wit.ai understanding and the out of scope parameter: 

Let's say the user says hello to my bot, right now I only have one intent that doesn't have anything to do with a greeting but the bot still thinks the user have this intent while saying hello.

In parallel there is the greetings traits that is rightfully understood.

And it is not possible to have no intents associated with a speech. 

I read in here that I should be able to train an utterance if there is an Intent OR a trait (as they are so called "first class citizens") however that is not the case..
I also read that the out of scope parameter ignore traits and entities...

So my question is do I need to create an intent that does what the greetings trait already does or is there a better way that i'm not getting ? Because I dont want my bot to say there is my intent in the "hello" text 

Thanks in advance :)

**Comment by Barbog** [comment]
Closing due to no movement on the issue. Please re-open or file a new task should the issue be persisting.
